config/routes.rb

```diff
@@ -9,6 +9,10 @@
     get "demo_partials/new"
     get "demo_partials/edit"
 
+    get "signup", to: "users#new"
+    post "signup", to: "users#create"
+
+    resources :users, only: %i(new create show)
```

Comment:
```suggestion
    resources :users, only: %i(show)
```
Chỗ này new, create mình dùng cho signup rồi, khai báo thêm sẽ thừa router